toolchain/check/handle_variable.cpp

```diff
@@ -32,15 +32,23 @@ auto HandleVariableInitializer(Context& context, Parse::NodeId parse_node)
 auto HandleVariableDecl(Context& context, Parse::NodeId parse_node) -> bool {
   // Handle the optional initializer.
   auto init_id = SemIR::InstId::Invalid;
-  bool has_init =
-      context.parse_tree().node_kind(context.node_stack().PeekParseNode()) !=
-      Parse::NodeKind::BindingPattern;
+  Parse::NodeKind next_kind =
+      context.parse_tree().node_kind(context.node_stack().PeekParseNode());
+  if (next_kind == Parse::NodeKind::ParamList) {
+    return context.TODO(parse_node, "tuple pattern in var");
+  }
+  bool has_init = next_kind != Parse::NodeKind::BindingPattern;
```

Comment:
Please add a TODO to find a more robust way to determine if there was an initializer.